src/python/WMComponent/JobSubmitter/JobSubmitterPoller.py

```diff
@@ -315,16 +315,25 @@ def refreshCache(self):
 
                 # all files in job have same location (in se names)
                 rawLocations = loadedJob["input_files"][0]["locations"]
+                
+                sitedb = SiteDB()
+ #               sitedb_names = set()
+                for l in rawLocations:
+#                    sitedb_names.update(sitedb.PNNtoPSN(l) or [])
+                    possibleLocations.update(sitedb.PNNtoPSN(l) or [])
```

Comment:
One problem getting the mapping from sitedb at this point, if sitedb goes down temporary or there is network problem.
The job submission will fail. I think we should get this information from wmbs_location_senames table instead.
We can create the memory cache as well we don't create call db every time.